command/agent/agent.go

```diff
@@ -419,13 +419,23 @@ func (a *Agent) loadKeyringFile(keyringFile string) error {
 // Stats is used to get various runtime information and stats
 func (a *Agent) Stats() map[string]map[string]string {
 	local := a.serf.LocalMember()
+	event_handlers := make(map[string]string)
+	var script_filter string
+
+	// Convert event handlres from a string slice to a string map
+	for _, script := range a.agentConf.EventScripts() {
+		script_filter = fmt.Sprintf("%s:%s", script.EventFilter.Event, script.EventFilter.Name)
```

Comment:
You could serialize `EventFilter` structure so consumers can avoid parsing the `:` later, but I'm ok with this as well since it's so simple, just occurred to me while looking at this.